actions/pack_action.go

```diff
@@ -30,10 +30,32 @@ type PackAction struct {
 	File             string
 }
 
+
+func (pf *PackAction) Verify(context *debos.DebosContext) error {
+
+	if len(pf.File) == 0 {
+		return fmt.Errorf("Please add a 'file' property.")
+	}
+
+	if len(pf.Compression) > 0 {
+		if pf.Compression != "gz" && pf.Compression != "xz" &&  pf.Compression != "bzip2" {
+			return fmt.Errorf("Unknown compression type: %s", pf.Compression)
+		}
+	}
+
+	return nil
+}
+
 func (pf *PackAction) Run(context *debos.DebosContext) error {
 	pf.LogStart()
 	outfile := path.Join(context.Artifactdir, pf.File)
+	opt := map[string]string{
```

Comment:
Please use map only for compression option instead of all passed options.
Also some versions of `tar` would be confused by option 'f' following the other option instead of file name.